public/scripts/main.js

```diff
@@ -127,12 +129,23 @@ const gameHandler = (function gameIIFE() {
     })
   }
 
+  function displayRole(role) {
+    document.querySelector('#image').src = role.image
+    document.querySelector('#name').textContent = role.name
+    document.querySelector('#description').textContent = role.description
```

Comment:
The general convention I was going for with these state handlers was selecting all the DOM elements into variables at [the top of the overall handler](https://github.com/ZakMiller/TheEmpire/pull/36/files#diff-63e210d669dcad8ddc23d1160a8c6187R102), but maybe that's only a good idea if they're used multiple times.

What you have here is probably fine for this case. What do you think?